packages/create-astro/src/index.ts

```diff
@@ -196,6 +228,36 @@ export async function main() {
 		await fs.promises.writeFile(pageFileLoc, newContent);
 	}
 
+	spinner.succeed();
+
+	const { name: pm } = whichPM() || { name: 'not_found' };
+	const supportedPM = ['npm', 'yarn', 'pnpm'].includes(pm);
+
+	if (supportedPM) {
+		spinner = ora({ color: 'green', text: 'Installing dependencies...' }).start();
```

Comment:
I think we intentionally decided not to auto-run these steps inside of `create-astro` to make maintenance more manageable, since it's not too much to ask users to do these things themselves.

The `peerDependencies` stuff above is amazing though, can we remove these "auto-run" additions for now and just focus on merging the peerDeps work?

Reply:
Done!